oci/config/convert/runtime.go

```diff
@@ -124,30 +128,26 @@ func MutateRuntimeSpec(g rgen.Generator, rootfs string, image ispec.Image, manif
 	g.SetProcessUID(uint32(execUser.Uid))
 	g.SetProcessGID(uint32(execUser.Gid))
 	g.ClearProcessAdditionalGids()
+
 	for _, gid := range execUser.Sgids {
 		g.AddProcessAdditionalGid(uint32(gid))
 	}
 	if execUser.Home != "" {
 		g.AddProcessEnv("HOME", execUser.Home)
 	}
 
+	// Set optional fields
+	var ports []string
+	for port := range image.Config.ExposedPorts {
+		ports = append(ports, port)
+	}
```

Comment:
I'd prefer it to return the `[]string` (the fact that internally it's implemented as a `map` is something that users shouldn't care about, but the fact it's a list of ports is important for implementations). In any case that's not important atm.